src/itkOMEZarrNGFFImageIO.cxx

```diff
@@ -135,7 +135,7 @@ OMEZarrNGFFImageIO::PrintSelf(std::ostream & os, Indent indent) const
   os << indent << "ChannelIndex: " << m_ChannelIndex << std::endl;
 }
 
-IOComponentEnum
+constexpr IOComponentEnum
 tensorstoreToITKComponentType(const tensorstore::DataType dtype)
 {
   switch (dtype.id())
```

Comment:
Interesting compile errors from Ubuntu/GCC, https://open.cdash.org/viewBuildError.php?buildid=9845800 at this particular `dtype.id()` call:

```
itkOMEZarrNGFFImageIO.cxx: In instantiation of 'constexpr const IOComponentEnum itk::toITKComponentType<float>':
itkOMEZarrNGFFImageIO.cxx:643:3:   required from here
itkOMEZarrNGFFImageIO.cxx:251:84:   in 'constexpr' expansion of 'itk::tensorstoreToITKComponentType(tensorstore::dtype_v<float>.tensorstore::StaticDataType<float>::operator tensorstore::DataType())'
itkOMEZarrNGFFImageIO.cxx:141:19:   in 'constexpr' expansion of 'dtype.tensorstore::DataType::id()'
itkOMEZarrNGFFImageIO.cxx:251:34: error: the value of 'tensorstore::internal_data_type::MakeDataTypeOperations<float>::operations' is not usable in a constant expression
```

Reply:
- Reported at https://github.com/google/tensorstore/issues/189

Reply:
Just fixed, by https://github.com/google/tensorstore/commit/91ea2a2fa4972a7b57fa63a72b81fd721d7a8429 😃

Reply:
Let's get #72 and #73 in before we try updating version of `tensorstore` we use. Or do you want to try that right away Niels?

Reply:
> Let's get #72 and #73 in before we try updating version of `tensorstore` we use.

Sounds like a plan 👍 

> Or do you want to try that right away Niels?

No, this one (#68) can wait a little longer, no problem!